sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -1929,6 +1929,12 @@ public CompletionStage<MutateRowResponse> writeRecord(KV<ByteString, Iterable<Mu
       return CompletableFuture.completedFuture(MutateRowResponse.getDefaultInstance());
     }
 
+    @Override
+    public CompletionStage<MutateRowResponse> writeSingleRecord(
+        KV<ByteString, Iterable<Mutation>> record) throws IOException {
```

Comment:
This is strange, why does this throw an IOException?
all of the IO should've happened async

Reply:
This throws an IO exception because all of the BigtableWriter interface throws IO exceptions